src/codegen/sys/cargo_toml.rs

```diff
@@ -186,7 +186,12 @@ fn fill_in(root: &mut Table, env: &Env) {
         let features = upsert_table(root, "features");
         let mut dox_features = Vec::new();
         for ext_lib in &env.config.external_libraries {
-            dox_features.push(Value::String(format!("{}/dox", ext_lib.crate_name)));
+            let crate_name = if ext_lib.crate_name == "gdk_pixbuf" {
```

Comment:
Should probably replicate the whole block here then to also cover cairo? And move it to a separate function :)

```rust
        let ext_package = if ext_lib.crate_name == "cairo" {
        let ext_package = if ext_lib.crate_name == "cairo" {
            format!("{}-sys-rs", ext_lib.crate_name)
            format!("{}-sys-rs", ext_lib.crate_name)
        } else if ext_lib.crate_name == "gdk_pixbuf" {
        } else if ext_lib.crate_name == "gdk_pixbuf" {
            "gdk-pixbuf-sys".into()
            "gdk-pixbuf-sys".into()
        } else {
        } else {
            format!("{}-sys", ext_lib.crate_name)
            format!("{}-sys", ext_lib.crate_name)
        };
```

Reply:
I don't think other renaming is necessary. 

1. The cairo dependency is renamed in the sys crates to just "cairo" (e.g in [pangocairo](https://github.com/gtk-rs/gtk-rs-core/blob/master/pangocairo/sys/Cargo.toml#L7)).
2. The gdk_pixbuf renaming is being done
3. the rest is just to add the -sys suffix but in the -sys crates, the dependencies are renamed and the suffix is not needed ([example](https://github.com/gtk-rs/gtk-rs-core/blob/master/pangocairo/sys/Cargo.toml#L11))

Since the gdk_pixbuf renaming is the only one that is needed for the dox feature, I decided it would be easiest to copy the code and not move it to a separate function. I can change that if you'd like me too though

Reply:
Maybe gdkpixbuf should also be renamed like cairo then so this special case here is not needed? Or is that not easily possible?

Reply:
As far as I can tell all other crates related to gtk use the - instead of _ in their names. I don't know why it was decided to do the renaming, but my guess is it was done to have a consistent naming scheme.

Reply:
Since it is generated and a dependency in the sys crates, I don't think the name is very important.

Reply:
Just to clarify, which name were you proposing instead of `gdk_pixbuf`?

Reply:
If we can avoid special casing gdk-pixbuf here that would be for the better for sure.